config/firewalld.service.in

```diff
@@ -17,6 +17,23 @@ StandardError=null
 Type=dbus
 BusName=org.fedoraproject.FirewallD1
 KillMode=mixed
+DevicePolicy=closed
+KeyringMode=private
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+ProtectClock=yes
+ProtectControlGroups=yes
+ProtectHome=yes
+ProtectHostname=yes
+ProtectKernelLogs=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+ProtectSystem=full
```

Comment:
firewalld writes its configuration to `/etc`. I don't think this will work.

Reply:
This is why I am not setting ProtectSystem=strict and instead using ProtectSystem=full which leaves out /etc/.  I will update the rest

Reply:
not according to `man systemd.exec`. With `ProtectSystem=full`, `/etc` is also read-only. I think it should be `ProtectSystem=yes`.

Reply:
Ah yes, I misread that.  Will update